apps/token-manager/contracts/TokenManager.sol

```diff
@@ -393,6 +393,10 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
         return tokens.sub(vestedTokens);
     }
 
+    function _spendableBalanceOf(address _holder) internal view returns (uint256) {
+        return _transferableBalance(_holder, now);
+    }
+
     function _transferableBalance(address _holder, uint256 _time) internal view returns (uint256) {
```

Comment:
Wondering whether this should always return the TM's token balance without iterating over its vestings. Even if there are current instances that have vestings for the TM, we should probably return the full balance as the vesting restrictions won't apply in practice.

If we did this we could also simplify the logic on `onTransfer(...)` as we wouldn't need to check for `_from == address(this)`, but it would perform an extra call to the token that can be avoided.

I would update this function anyway so the `spendableBalanceOf ` getter returns the actual spendable balance of the TM.

Reply:
> If we did this we could also simplify the logic on onTransfer(...) as we wouldn't need to check for _from == address(this), but it would perform an extra call to the token that can be avoided.

Even though this is a little bit more expensive for doing `assign()` and `assignVested()`, I think this makes sense as it cheapens other, arbitrary token transfers.

It also makes the branching simpler, which is always good 😄